listenbrainz_spark/stats/user/listening_activity.py

```diff
@@ -25,6 +25,26 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_quarter_offset(_date: date) -> relativedelta:
```

Comment:
it's not immediately clear to me the reason for having both this method and the "1 quarter back" method in `listenbrainz_spark/stats/__init__.py`. At the very least, the name for this method is confusing without reading the docstring.

Reply:
I'll improve the docs and method name. The reason is that listening activity stats needs data for last two time periods. So that the frontend can compare those. Like listening activity generates the data for 2 weeks and frontend presents it as a comparision of each weekday of the two weeks.

Reply:
Updated doc for this method. Let me know if I should add more details.